### ISSUE TITLE
neo.mjs Website

### ISSUE THREAD

**Issue by tobiu** [neomjs/neo#712]
I think neo.mjs reached the point were the online examples are no longer sufficient.

**What I have in mind:**
1. We need a logo (#7) 
2. An easy to understand framework intro (could use help on this one, I am way too deep into it)
3. We need a basic design how it should look like
4. The website should get build using neo.mjs (kind of obvious)
5. The docs app should get integrated (maybe in a tab?)
6. Blogs & tutorials should get their own section (tab as well?)
7. The online examples should get integrated, but in a nicer way (maybe a card with a preview pic for each one?)
8. I would like a separate section to explain the BaaS model => willing to set up a team for trainings, professional services & support in case the framework got traction and you need this.

**My current todo list:**
Right now I am working on the shared workers based covid app and want to really polish this one. Still some work on the framework core left to make this happen. Afterwards I would like to create a video to showcase it & another Medium Blog Post.

Afterwards I still need to finish Part 2 of the Covid App tutorial.

I could create a Part 3 on how to transform the Covid App into the version which can run inside multiple Browser Windows, in case this is helpful for you (need your input on this one).

There are still a lot of other items left:
* Support for Drag&Drop
* Dialogs (resizable, draggable)
* Supporting dynamic imports for the dist env (App worker scope)
* Enhancing the framework for better mobile support (especially adjusting some of the example apps)
* Buffered Stores & Views (e.g. the Grid implementation)

Just to name a few.

In other words: I could use your help on this one :)

Thanks & best regards,
Tobi
